Cannot use console / vnc with virtctl

<!-- This form is for bug reports and feature requests ONLY!
Also make sure that you visit our User Guide at https://kubevirt.io/user-guide/
-->

/kind bug


**What happened**:
When running `virtctl console dors-vm` i'm getting the following error :
```
panic: assignment to entry in nil map

goroutine 52 [running]:
net/textproto.MIMEHeader.Set(...)
	GOROOT/src/net/textproto/header.go:22
net/http.Header.Set(...)
	GOROOT/src/net/http/header.go:37
kubevirt.io/kubevirt/vendor/k8s.io/client-go/plugin/pkg/client/auth/exec.(*roundTripper).RoundTrip(0xc000114300, 0xc00026c200, 0x7ffeefbffaf0, 0x7, 0x25184e8)
	vendor/k8s.io/client-go/plugin/pkg/client/auth/exec/exec.go:307 +0x435
kubevirt.io/client-go/kubecli.asyncSubresourceHelper.func1(0x2710c80, 0xc000114300, 0xc00026c200, 0xc00042c660)
	staging/src/kubevirt.io/client-go/kubecli/async.go:48 +0x50
created by kubevirt.io/client-go/kubecli.asyncSubresourceHelper
	staging/src/kubevirt.io/client-go/kubecli/async.go:46 +0x3b7
``` 
**What you expected to happen**:
Connect the VM

```
kubectl get vmis
NAME      AGE    PHASE     IP            NODENAME                       READY
dors-vm   123m   Running   10.120.2.99   ip-10-120-2-127.ec2.internal   True
```
**Environment**:
- KubeVirt version (use `virtctl version`):  
```
Client Version: version.Info{GitVersion:"v0.46.0", GitCommit:"3ebb0a03fde639ea9afb5ee83ac7ac72dbeb4a9d", GitTreeState:"clean", BuildDate:"2021-10-08T21:20:46Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{GitVersion:"v0.46.0-dirty", GitCommit:"3ebb0a03fde639ea9afb5ee83ac7ac72dbeb4a9d", GitTreeState:"dirty", BuildDate:"2021-10-08T21:39:28Z", GoVersion:"go1.16.6", Compiler:"gc", Platform:"linux/amd64"}
```

- Kubernetes version (use `kubectl version`):
```
Client Version: version.Info{Major:"1", Minor:"21", GitVersion:"v1.21.5", GitCommit:"aea7bbadd2fc0cd689de94a54e5b7b758869d691", GitTreeState:"clean", BuildDate:"2021-09-15T21:10:45Z", GoVersion:"go1.16.8", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20+", GitVersion:"v1.20.7-eks-d88609", GitCommit:"d886092805d5cc3a47ed5cf0c43de38ce442dfcb", GitTreeState:"clean", BuildDate:"2021-07-31T00:29:12Z", GoVersion:"go1.15.12", Compiler:"gc", Platform:"linux/amd64"}
```

--- a/staging/src/kubevirt.io/client-go/kubecli/vmi.go
+++ b/staging/src/kubevirt.io/client-go/kubecli/vmi.go
@@ -151,6 +151,7 @@ func RequestFromConfig(config *rest.Config, resource, name, namespace, subresour
 	req := &http.Request{
 		Method: http.MethodGet,
 		URL:    u,
+		Header: map[string][]string{},
 	}
 
 	return req, nil

